apollo-federation/src/sources/connect/json_selection/parser.rs

```diff
@@ -1020,7 +1020,7 @@ pub fn parse_string_literal(input: Span) -> IResult<Span, WithRange<String>> {
 
 #[derive(Debug, PartialEq, Eq, Clone, Default)]
 pub struct MethodArgs {
-    pub(super) args: Vec<WithRange<LitExpr>>,
+    pub(crate) args: Vec<WithRange<LitExpr>>,
```

Comment:
I don't see an immediate problem here (compared to `PathList`), but I would note that the `WithRange` representation was relatively recently added and probably should not be considered stable yet.

Reply:
I'm fine with it being unstable and the validation code needing to change. There is significant value here for customers if we can get something in relatively soon - being able to point customers to a specific range associated with a validation error (such as an arrow method) is a way better experience that just highlighting the entire selection.